examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.cpp

```diff
@@ -787,6 +743,7 @@ void HandleDHCPPolling()
      */
     if ((ip6_addr_ispreferred(netif_ip6_addr_state(sta_netif, 0))) && !hasNotifiedIPV6)
     {
+        ChipLogProgress(DeviceLayer, "SLAAC OK: linklocal addr: %s", ip6addr_ntoa(netif_ip6_addr(sta_netif, 0)));
```

Comment:
ip6addr_ntoa is not thread safe. Strongly recommend using an alternative.

Reply:
Replaced wih `ip6addr_ntoa_r`